lib/Transforms/LowerPolynomialEval/LowerPolynomialEval.cpp

```diff
@@ -28,16 +28,20 @@ struct LowerPolynomialEval
     if (method == PolynomialApproximationMethod::Automatic) {
       patterns.add<LowerViaHorner, LowerViaPatersonStockmeyerChebyshev,
                    LowerViaPatersonStockmeyerMonomial>(context,
-                                                       /*force=*/false);
+                                                       /*force=*/false,
+                                                       minCoefficientThreshold);
```

Comment:
On second thought, should I even allow dropping coefficients on monomial-basis methods?